<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>410</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>361</width>
    <height>410</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>361</width>
    <height>410</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Skander Calc</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>26</y>
     <width>361</width>
     <height>361</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="lineEdit" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>10</y>
      <width>341</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <family>Arab</family>
      <pointsize>25</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="focusPolicy" >
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="layoutDirection" >
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text" >
     <string>0</string>
    </property>
    <property name="maxLength" >
     <number>16</number>
    </property>
    <property name="readOnly" >
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_c" >
    <property name="geometry" >
     <rect>
      <x>150</x>
      <y>70</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <family>Sans Serif</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy" >
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text" >
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ce" >
    <property name="geometry" >
     <rect>
      <x>80</x>
      <y>70</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy" >
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text" >
     <string>CE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plus" >
    <property name="geometry" >
     <rect>
      <x>290</x>
      <y>310</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <family>Sans Serif</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy" >
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text" >
     <string>+</string>
    </property>
    <property name="autoDefault" >
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_11" >
    <property name="geometry" >
     <rect>
      <x>220</x>
      <y>250</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy" >
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text" >
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_minus" >
    <property name="geometry" >
     <rect>
      <x>290</x>
      <y>250</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <family>Sans Serif</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy" >
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text" >
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>190</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy" >
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text" >
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_bcksp" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>70</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy" >
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text" >
     <string>BCKSP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_div" >
    <property name="geometry" >
     <rect>
      <x>290</x>
      <y>190</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <family>Sans Serif</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy" >
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text" >
     <string>÷</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5" >
    <property name="geometry" >
     <rect>
      <x>80</x>
      <y>190</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy" >
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text" >
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mult" >
    <property name="geometry" >
     <rect>
      <x>290</x>
      <y>130</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <family>OpenSymbol</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy" >
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text" >
     <string>x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3" >
    <property name="geometry" >
     <rect>
      <x>150</x>
      <y>250</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy" >
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text" >
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>310</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy" >
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text" >
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sqrt" >
    <property name="geometry" >
     <rect>
      <x>220</x>
      <y>190</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <family>Sans Serif</family>
      <pointsize>23</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy" >
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text" >
     <string>√x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7" >
    <property name="enabled" >
     <bool>true</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>130</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy" >
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text" >
     <string>7</string>
    </property>
    <property name="autoDefault" >
     <bool>true</bool>
    </property>
    <property name="default" >
     <bool>false</bool>
    </property>
    <property name="flat" >
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_min_plus" >
    <property name="geometry" >
     <rect>
      <x>290</x>
      <y>70</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <family>Sans Serif</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy" >
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text" >
     <string>±</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_x_pwr_2" >
    <property name="geometry" >
     <rect>
      <x>220</x>
      <y>130</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <family>Sans Serif</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy" >
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text" >
     <string>x²</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2" >
    <property name="geometry" >
     <rect>
      <x>80</x>
      <y>250</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy" >
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text" >
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_equal" >
    <property name="geometry" >
     <rect>
      <x>220</x>
      <y>310</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <family>Sans Serif</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy" >
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text" >
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9" >
    <property name="geometry" >
     <rect>
      <x>150</x>
      <y>130</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy" >
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text" >
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8" >
    <property name="geometry" >
     <rect>
      <x>80</x>
      <y>130</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy" >
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text" >
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6" >
    <property name="geometry" >
     <rect>
      <x>150</x>
      <y>190</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy" >
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text" >
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1byx" >
    <property name="geometry" >
     <rect>
      <x>220</x>
      <y>70</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <family>Sans Serif</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy" >
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text" >
     <string>1/x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>250</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font" >
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy" >
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text" >
     <string>1</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>361</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>File</string>
    </property>
    <addaction name="action_Close" />
    <addaction name="action_Hide" />
   </widget>
   <widget class="QMenu" name="menuEdit" >
    <property name="title" >
     <string>Edit</string>
    </property>
    <addaction name="separator" />
    <addaction name="action_Copy" />
    <addaction name="action_Paste" />
   </widget>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>Help</string>
    </property>
    <addaction name="action_About" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuEdit" />
   <addaction name="menuHelp" />
  </widget>
  <widget class="QStatusBar" name="statusbar" >
   <property name="enabled" >
    <bool>true</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>387</y>
     <width>361</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <action name="action_Close" >
   <property name="text" >
    <string>Close</string>
   </property>
  </action>
  <action name="action_Hide" >
   <property name="text" >
    <string>Hide</string>
   </property>
  </action>
  <action name="action_Copy" >
   <property name="text" >
    <string>Copy</string>
   </property>
  </action>
  <action name="action_Paste" >
   <property name="text" >
    <string>Paste</string>
   </property>
  </action>
  <action name="action_About" >
   <property name="text" >
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
